### PR TITLE
Merge in cisagov/skeleton-tf-module and update code for use in the COOL

### PR DESCRIPTION
**Pull request by jsf9k**
# 

## 🗣 Description

In this pull request I merge in cisagov/skeleton-tf-module and update code for use in the COOL.  The latter mostly involves turning the repo into a full-fledged Terraform role.

## 💭 Motivation and Context

The `dmarc.cyber.dhs.gov` email domain is only verified in the COOL DNS account.  Therefore the dmarc-import application must be moved there.

## 🧪 Testing

I have used these changes to successfully deploy dmarc-import to the COOL DNS account.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+# dmarc-import-terraform #
 
-[![Build Status](https://travis-ci.com/cisagov/dmarc-import-terraform.svg?branch=develop)](https://travis-ci.com/cisagov/dmarc-import-terraform)
+[![GitHub Build Status](https://github.com/cisagov/dmarc-import-tf-module/workflows/build/badge.svg)](https://github.com/cisagov/dmarc-import-tf-module/actions)
 
-`dmarc-import-terraform` contains the Terraform configuration files
-that build the AWS infrastrusture used for parsing DMARC aggregate
-reports.  This repository goes along with
+`dmarc-import-terraform` contains the Terraform configuration files to
+build the AWS infrastrusture used for parsing DMARC aggregate reports.
```
Comment (dav3r) [inline README.md]: It's not your change, but `infrastructure` is misspelled here.

File: README.md [hunk 2]
```diff
@@ -13,30 +13,70 @@ Here is a [Cloudcraft.co](https://cloudcraft.co) diagram of the basic
...
+  queue_name                = var.queue_name
+  rule_set_name             = var.rule_set_name
+  tags                      = var.tags
+  temporary_bucket_name     = var.temporary_bucket_name
+}
 ```
 
+## Examples ##
+
+No examples.
```
Comment (dav3r) [inline README.md:43]: Can you add an issue to create an example here?

File: iam.tf [hunk 3]
```diff
@@ -112,13 +112,13 @@ data "aws_iam_policy_document" "cloudwatch_doc" {
...
-  role   = aws_iam_role.role.id
-  policy = data.aws_iam_policy_document.cloudwatch_doc.json
+  role   = aws_iam_role.lambda.id
+  policy = data.aws_iam_policy_document.cloudwatch_lambda.json
 }
 
 # IAM policy document that that allows the Lambda function to invoke
 # itself.  This will be applied to the role we are creating.
-data "aws_iam_policy_document" "lambda_doc" {
+data "aws_iam_policy_document" "lambda_lambda" {
```
Comment (dav3r) [inline iam.tf:121]: It's lambdas all the way down! 😄

**Review by dav3r — APPROVED** [review]
So fresh, clean, and modern - strong work! 💪

File: README.md (hunk 2 above)
Comment (jsf9k) [reply README.md:43]: See #12.

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md]: See commit baf88ac.

**Review by mcdonnnj — APPROVED** [review]
So fresh, so modern.

File: iam.tf (hunk 3 above)
Comment (felddy) [reply iam.tf:121]: `Lambda` `Lambda` `Lambda`  and `OMEGA MU`

https://www.youtube.com/watch?v=5sqc98KHT4c

**Review by felddy — APPROVED** [review]
Looks great.  Not thrilled about this landing in the DNS account, but that is the world we live in.  Make sure you don't miss the video link: https://www.youtube.com/watch?v=5sqc98KHT4c

```
Clap your hands everybody, and everybody clap your hands!
We're Lambda Lambda Lambda and... Omega Mu
We come here on stage tonight to do a show for you
We've got a rockin rhythm and a high tech sound
That'll make you move your body down to the ground
```

**Pull request merged**